examples/cs/comm/pingpong/PingPong.cs

```diff
@@ -31,7 +32,9 @@ public static void Main()
 
             Shutdown(transport);
 
-            Console.WriteLine("Done with all requests.");
+            // Let everything else finish so we can print last.
+            Thread.Sleep(50);
```

Comment:
There's some part of `Shutdown(transport)` that keeps running after the shutdown tasks are completed?